app/src/main/java/org/inspire/breath/activities/Understanding_results.java

```diff
@@ -0,0 +1,59 @@
+package org.inspire.breath.activities;
+
+import android.content.Intent;
+import android.se.omapi.Session;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import org.inspire.breath.R;
+import org.inspire.breath.data.AppRoomDatabase;
+import org.inspire.breath.data.Recording;
+import org.inspire.breath.data.blobs.MalariaTestResult;
+
+public class Understanding_results extends AppCompatActivity {
+
+    private Recording session ;//session object
+    private int age;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_understanding_results);
+        session = AppRoomDatabase.getDatabase().recordingDao().getRecordingById(HomeActivity.currentSession).get(0);//getting session instance --> getting info
```

Comment:
You'll need to rebase and fix any compile issues, code has changed since you wrote this

Reply:
marking as unresolved, need to update new code (no longer need to use .get(0)